actors/miner/src/lib.rs

```diff
@@ -4176,15 +4176,19 @@ fn request_terminate_deals(
     const MAX_LENGTH: usize = 8192;
 
     for chunk in deal_ids.chunks(MAX_LENGTH) {
-        rt.send(
+        // Intentionally swallow this error to prevent frozen market cron corruption from
+        // also freezing this miner cron.
+        // This is safe from a malicious caller perspective because this method's callstack
+        // should always originate with the trusted system caller.
```

Comment:
This method is reached from outside callers by by terminate_sectors -> process_early_terminations.

Reply:
thx will fix

Reply:
Maybe we should change this to check that the caller is cron before swallowing the error?